libraries/chain/include/eosio/chain/combined_database.hpp

```diff
@@ -76,17 +72,13 @@ namespace eosio { namespace chain {
 
     private:
       std::unique_ptr<chainbase::database::session> cb_session    = {};
-      eosio::session::undo_stack<rocks_db_type>*     kv_undo_stack = nullptr;
    };
 
    class combined_database {
```

Comment:
This should be removed along with the combined session. Basically the code should look like it does in `release/2.0.x`

Reply:
as discussed in DM, we just renamed this to `kv_session` and `kv_database` as this was refactoring

Reply:
update: `combined_session` now has old name `maybe_session` and combined_database transformed into namespace `db_utils`